### ISSUE TITLE
Logo for the vscode-extension

### ISSUE THREAD

**Issue by Xenap** [EverMCServer/vscode-BetonQuest#4]
As there isn't a logo yet for the extension i propose a temporary one, made by myself

![book_logo](https://github.com/EverMCServer/vscode-BetonQuest/assets/66042272/521af371-d7a5-411c-a014-937c4428bc97)

**Comment by Xenap** [comment]
Do you plan on using another logo, or do you have something different in mind?

**Comment by k-jiang** [comment]
Sorry for the late reply. I have been very busy recently.

Thank you for your work on the logo! But I do have some different ideas about it. I want the logo to be more reflected on features the plugin has, like flowcharts. And it would be great if the art style can be more stood-out while closed to what [the official logo](https://github.com/BetonQuest/BetonQuest/blob/main/docs/_media/brand/Logo/LogoChainless1K.png) looks like.

I'm not good at logo design. I have tried achieve it with AI like Stable Diffusion but the result is hard to be satisfied. Perhaps Midjourney might yield a better result but I don't have time for it. Or I may hire a designer in the future.

The project is still on its primitive stage. Plus the VSCode does not like switching logo too often. So I decided to just keep it "blank" for this moment until I have more resources.
